modules/claims_api/app/controllers/claims_api/v1/forms/power_of_attorney_controller.rb

```diff
@@ -30,7 +30,7 @@ def submit_form_2122 # rubocop:disable Metrics/MethodLength
           validate_poa_code!(poa_code)
           validate_poa_code_for_current_user!(poa_code) if header_request? && !token.client_credentials_token?
           file_number = check_file_number_exists!
-          dependent_participant_id, claimant_ssn = validate_dependent_claimant!(poa_code:)
+          claimant_information = validate_dependent_claimant!(poa_code:)
```

Comment:
We need to include first and last name now as well so just creating the whole object and adding it to the headers to get in the dependent job.

Reply:
Same as below: passing the service object through the jobs eliminates the auth header munging.